### DISCUSSION TITLE
Add feature for create parent title in KPI chart

### DISCUSSION THREAD

**Discussion by siddizi** [chartbrew/chartbrew#262] [Ideas]
I have a several charts which have same parent label but different child. I cant group it and I have a very long string in my chart(not friendly). See pic
![image](https://github.com/user-attachments/assets/ed63fd7a-e79f-4441-a67c-45b0941853b5)

Whad do you think about group it? See pic for example
![image](https://github.com/user-attachments/assets/8d35793c-35b1-49fd-bd0c-a3f0ef94cd90)

**Comment by razvanilin** [comment]
Hey @siddizi sorry I missed your post!

This seems to be a visualization that should be better set up with a tree view rather than metric view like this. In this case I would split the chart into two instead of using one for multiple parents. The first one will be for Parent 1 and the 2nd for Parent 2.

**Reply by siddizi** [reply]
Hey @razvanilin! 
Yes, I agree with you.
